lib/web/apiserver.go

```diff
@@ -1249,6 +1251,36 @@ func (h *Handler) getResetPasswordToken(ctx context.Context, tokenID string) (in
 	}, nil
 }
 
+// createNodeJoinToken creates a node join token and sends it back along with its expiry.
+// A node join token is used to join a node to a cluster.
+func (h *Handler) createNodeJoinToken(w http.ResponseWriter, r *http.Request, _ httprouter.Params, ctx *SessionContext) (interface{}, error) {
+	clt, err := ctx.GetClient()
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	defaultTTL := "4h"
+	ttl, err := time.ParseDuration(defaultTTL)
```

Comment:
Move this constant to https://github.com/gravitational/teleport/blob/master/lib/defaults/defaults.go 
and use `time.Time` type to define it's value

No need to do time.ParseDuratiion(..)